swagger.yml

```diff
@@ -312,7 +312,7 @@ paths:
   '/tyk/apis/oas':
     get:
       description: |-
-         List APIs, in OAS format (only APIs saved in OAS format)
+         List APIs, in OAS format
```

Comment:
I think this part is important `(only APIs saved in OAS format)` - suppose we have 10 APIs saved and only 30 of them are in OAS format(migrated/created), only 30 APIs will be listed with this endpoint